### PR TITLE
[Fire LPP-56152] LPD-39828

### PR DESCRIPTION
**Pull request by vngoliveira**
[LPP-56152](https://liferay.atlassian.net/browse/LPP-56152)
[LPD-39828](https://liferay.atlassian.net/browse/LPD-39828)

[LPP-56152]: https://liferay.atlassian.net/browse/LPP-56152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LPD-39828]: https://liferay.atlassian.net/browse/LPD-39828?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by vngoliveira** [comment]
ci:test:sf

**Comment by vngoliveira** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/recipient/RoleNotificationRecipientBuilder.java [hunk 1]
```diff
@@ -121,6 +121,29 @@ protected void deactivate() {
 		_serviceTrackerList.close();
 	}
 
+	protected boolean isSelfAssignedUser(
```
Comment (linolaoj) [inline modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/recipient/RoleNotificationRecipientBuilder.java:124]: Hi @vngoliveira , I understand that this method was turned `protected` only to fit in the test. If that is the case we can use the `Method` class like what is done in `ObjectEntryEntityModelTest.java` to keep this method `private`.

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Comment by vngoliveira** [comment]
ci:test:sf

**Review by linolaoj — APPROVED** [review]

**Comment by vngoliveira** [comment]
ci:test:workflow

**Comment by vngoliveira** [comment]
ci:test:relevant

**Comment by linolaoj** [comment]
ci:forward

**Pull request closed without merging**